lib/HTML/Directives/Wrap.php

```diff
@@ -42,6 +43,11 @@ public function processSub(
         string $data,
         array $options
     ): ?Node {
+        // if there is a "Wrap" directive (e.g. a note::), blank content is unexpected
+        if ($document === null) {
+            throw new RuntimeException('Content expected, none found.');
+        }
```

Comment:
In practice, from what I can see, this `Wrap` directive is used for the `note::` directive only. Most directives do no allow NO content. Note is one of those. This behavior is consistent with Sphinx (even the wording). Thanks to the error handling in `DocumentParser`, the user sees an error with the name of the file and approximate line number.